server/views/pages/startPPCS.njk

```diff
@@ -2,19 +2,33 @@
 
 {% set pageTitle = makePageTitle({ pageHeading: "Start page for PPCS" }) %}
 {% block beforeContent %}
-  {% if notification.isVisible %}
-    {% set html %}
-      {% if notification.header %}
-        <p class="govuk-notification-banner__heading">
-          {{ notification.header }}
-        </p>
-      {% endif %}
-      <span class="govuk-body">{{ notification.body }}</span>
-    {% endset %}
-    {{ govukNotificationBanner({html: html}) }}
-  {% endif %}
+    {% if notification.isVisible %}
+        {% set html %}
+            {% if notification.header %}
+                <p class="govuk-notification-banner__heading">
+                    {{ notification.header }}
+                </p>
+            {% endif %}
+            <span class="govuk-body">{{ notification.body }}</span>
+        {% endset %}
+        {{ govukNotificationBanner({html: html}) }}
+    {% endif %}
 {% endblock %}
 {% block content %}
+    {% if not validMappingAndPpudUser %}
```

Comment:
There seems to be a pattern used elsewhere for displaying errors like this which uses the `error-summary.njk` template, which in turn relies on `res.locals` having an `errors` field. I haven't looked much further than that, though, do you know anything about it? Is it worth following that pattern if it can accommodate what we need?

Reply:
I'll look into the validation controller pattern in the next commit